### ISSUE TITLE
Wrong results for similarly named movie(released in different years) and won't take new higher quality uploads

### ISSUE THREAD

**Issue by BRKsays** [mhdzumair/MediaFusion#258]
I searched for a movie in Stremio released in 2024 but the results include a movie with the same name but released in 2013. It also includes a Pete Davidson Comedy show, another couple of movies released in 2010 and 2015 with one or two words added to the 2024 movie title.

I have Torrentio & KnightCrawler too but only Media Fusion is giving back results for the movie.

The result lists contain only a PreDVDrip quality for the movie I want whereas 4K WEB-DL is available.

When I tried to upload the 4K WEB-DL torrent file it says "Torrent already exists and attached to meta id: xxxxxxxx".

Can we flag wrongly listed movies and upload newer and better qualities even though there exists previously uploaded files?

**Comment by mhdzumair** [comment]
Yup i had that idea in my mind. We can setup approval process to setting up those data in case of wrongly handled. Will do this.

**Comment by mhdzumair** [comment]
This is now supported in case if the torrent is attached to a different meta id.
